client/packages/common/src/ui/layout/tables/columns/types.ts

```diff
@@ -19,6 +19,7 @@ export interface CellProps<T extends RecordWithId> {
   localisedText: TypedTFunction<LocaleKey>;
   localisedDate: (date: string | number | Date) => string;
   debounceTime?: number;
+  dense?: boolean;
```

Comment:
A lot of our `custom` cells aren't formatting correctly when the table is dense 👀